department-of-veterans-affairs/va.gov-cms
# VACMS-2041 - Update office content model with Connect with Us

## Description

Closes #2041 .

## Testing done
Added an office locally and verified new field existence on node create page. 

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/140395084-3674a439-ef2a-4213-99f0-6cfc6cf85fd2.png)


## QA steps

As a cms editor I can 
1. View the node edit/create screen for an office (/node/38442/edit) and see the new fields for social media links
2. View the node edit/create screen for an office (/node/38442/edit) and see the new fields for external link
3. View the node edit/create screen for an office (/node/38442/edit) and see the new fields for newsletter link
4. View the node edit/create screen for an office (/node/38442/edit) and see that the fields have been organized into vertical tabs
4. View the form view screen for a benefits hub (/admin/structure/types/manage/landing_page/form-display) and confirm the new optional field 'Related office' is hidden
5. View the  form view screen for a campaign landing page (/admin/structure/types/manage/campaign_landing_page/form-display) and confirm the new optional field 'Related office' is hidden
6. Then validate Acceptance Criteria from issue

- [x] Add social media links field to office nodes
- [x] Add email updates field to office nodes
- [x] Add external link field to office nodes
- [x] Add an optional entity reference to Office from Benefits Hub Landing page (this is to support the Records Hub)
- [x] Add an optional entity reference to Office from Campaign Landing page (this will replace the existing Connect With Us field)
- [x] Update drupal spec tool for new fields
- [x] Hide new fields until frontend has been implemented

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
